Assets/Mirror/Runtime/Transport/Transport.cs

```diff
@@ -54,7 +54,7 @@ public virtual bool Available()
 
         // common
         public abstract void Shutdown();
-        public abstract int GetMaxPacketSize(int channelId=Channels.DefaultReliable);
```

Comment:
I think that is some kind of coding standard, people often use no space for = in default parameters

Reply:
NetworkConnection:137 NetworkServer:189 both uses spaces around =. I've never seen that coding standard you mention, but if you think we should stick to it, lets be consistent and do it across the whole codebase. There were less instances of no spacing though, so i thought it was just a typo.